pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -1632,6 +1639,35 @@ public static Multimap<String, Metric> parseMetrics(String metrics) {
         return parsed;
     }
 
+    @Test
+    public void testRawMetricsProvider() throws IOException {
+        PrometheusMetricsProvider rawMetricsProvider = new PrometheusMetricsProvider();
+        try {
+            rawMetricsProvider.start(new PropertiesConfiguration());
+            rawMetricsProvider.getStatsLogger("test_raw")
+                .scopeLabel("topic", "persistent://public/default/test-v1")
+                .getOpStatsLogger("writeLatency")
+                .registerSuccessfulEvent(100, TimeUnit.NANOSECONDS);
+
+            getPulsar().addPrometheusRawMetricsProvider(rawMetricsProvider);
+            HttpClient httpClient = HttpClientBuilder.create().build();
+            final String metricsEndPoint = getPulsar().getWebServiceAddress() + "/metrics";
+            HttpResponse response = httpClient.execute(new HttpGet(metricsEndPoint));
+            Multimap<String, Metric> metrics = parseMetrics(EntityUtils.toString(response.getEntity()));
+            if (((List<Metric>) metrics.get("test_raw_writeLatency_count"))
```

Comment:
Let's extract to variable `(List<Metric>) metrics.get("test_raw_writeLatency_count"))
                .get(0)` to make it easier to read the next lines